### PR TITLE
Constant value simplification - GameParser

### PR DESCRIPTION
**Pull request by DanVanAtta**
## Overview
IDE analysis refactor to remove inline constant value'd parameters.

## Functional Changes
none, refactor

### CODE REVIEW

File: game-core/src/main/java/games/strategy/engine/data/GameParser.java [hunk 1]
```diff
@@ -495,9 +494,9 @@ private Element getSingleChild(final String name, final Node node, final boolean
         .collect(Collectors.toList());
   }
 
-  private static List<Node> getNonTextNodesIgnoring(final Node node, final String ignore) {
+  private static List<Node> getNonTextNodesIgnoring(final Node node) {
```
Comment (RoiEXLab) [inline game-core/src/main/java/games/strategy/engine/data/GameParser.java]: I would've renamed to method to `getNonTextNodesIgnoringValue` or something

**Comment by DanVanAtta** [comment]
I've a branch that has similar changes for 50 more files : (

I've sat on that for too long, the fact that these issues are coming up in other PRs indicates that.

File: game-core/src/main/java/games/strategy/engine/data/GameParser.java (hunk 1 above)
Comment (DanVanAtta) [reply game-core/src/main/java/games/strategy/engine/data/GameParser.java]: Updated the method name per suggestion: 95f4abd8d

**Comment by DanVanAtta** [comment]
> Unless you want to move the `throws GameParseException` into the same line as the method declaration for many methods, feel free to merge

Took a look at this post-merge intending to follow-up. Looks like most would hit the line length limit if made to be a single line. This suggestion I think indicates we should get moving on google-java-format which would take care of this for us.

**Pull request merged**